Dockerfile

```diff
@@ -0,0 +1,79 @@
+# Use Ubuntu 16.04 LTS as the basis for the Docker image.
```

Comment:
let's update this and also maybe add a leading comment block with how the  image should be used